guides/guide-for-adding-windows-node.md

```diff
@@ -94,10 +94,12 @@ curl -L https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/mast
 > `kubeadm version`
 
 5. Apply kube-flannel-rbac.yml from sig-windows-tools/kubeadm/flannel
+
+Modify the `net-conf.json` section of the flannel manifest in order to set the VNI to 4096 and the Port to 4789 here as well.
 Next you will need to apply the configuration that allows flannel to spawn pods and keep them running:
 
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/hostprocess/flannel/flanneld/kube-flannel-rbac.yml
+curl -L https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/hostprocess/flannel/flanneld/kube-flannel-rbac.yml | sed s'/"Type": "vxlan"/"Type": "vxlan", "VNI": 4096, "Port": 4789/' | kubectl apply -f -
```

Comment:
Since we own this file can we just manually added it to the yaml?  It will almost always be these values.

Reply:
I agree, for the time being we only support VXLAN connectivity for flannel on windows. Once we also support host-gw as second option (which is valid for flannel on windows) we should anyway remove the net-conf.json configmap from kube-flannel-rbac.yaml because the config-map is installed together with the linux-side flannel installation already (but in another namespace). I will update the pull-request and try to fix the cla issue as suggested.